app/lib/index.js

```diff
@@ -68,8 +68,13 @@ global.modelerDirectory = appPath;
 
 var menu = new Menu(process.platform);
 
+// bootstrap filesystem
+var fileSystem = new FileSystem({
+  dialog: dialog
```

Comment:
File system doesn't depend on dialog anymore.

Reply:
Separate issue?

Reply:
Or simple quick fix?

Reply:
I'll fix it. 👍